src/ontology/cl-edit.owl

```diff
@@ -29470,6 +29471,21 @@ SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000005444))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
 
+# Class: obo:CL_4023122 (oxytocin receptor sst GABAergic cortical interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) obo:IAO_0000115 obo:CL_4023122 "An interneuron located in the cerebral cortex that express the oxytocin receptor. These interneurons also express somatostatin.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) obo:RO_0002175 obo:CL_4023122 obo:NCBITaxon_10090)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023122 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_4023122 "OxtrINs")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) rdfs:comment obo:CL_4023122 "These cells were first identified in the mouse.")
```

Comment:
@shawntanzk "in mouse" instead of "in the mouse" sounds slightly better to me, but it's ~6pm and I'm starting to see only trees and ignore the forest, so I'll stop here - you're good to merge! Thank you.

Reply:
yeah definitely not with the - my mind was thinking in the taxon >.< lol - I need Grammarly for protege :p lol thanks for all the catches!